weaviate/client.py

```diff
@@ -90,7 +90,7 @@ def __init__(
                 {"X-OpenAI-Api-Key": "<THE-KEY>"}, {"X-HuggingFace-Api-Key": "<THE-KEY>"}
             by default None
         wait_for_weaviate : int or None
-            How long the client will wait for weaviate to start before raising a ConnectionError.
+            How long the client will wait for weaviate to start before raising the appropiate Error,
```

Comment:
just spell the error out, otherwise good from my side

Reply:
The issue is that it does not necessarily raise that particular error. If an invalid URL is given for example another error is raised.

Reply:
I just realized that it does not wait for that time then.

Reply:
Nevermind.